test/sanity/language/exceptions/catchingErrors.wtest

```diff
@@ -28,4 +28,16 @@ describe "catching errors" {
     assert.equals(20, cuenta.sacar(10))
   }
 
+  test "try expression should be usable as message argument, even with multiline body" {
+    const o = object {
+      method add(a, b, c) = a + b + c
+    }
+
+    assert.equals(6, o.add(1, try { 0 ; 2 } catch e { 0 }, 3))
+    assert.equals(6, o.add(1, try { 0 ; 2 } then always { 0 }, 3))
+    assert.equals(6, o.add(1, try { 0 ; 2 } catch e { 0 } then always { 0 }, 3))
+    assert.equals(6, o.add(1, try { 0; throw new Exception() } catch e { 0 ; 2 }, 3))
+    assert.equals(6, o.add(1, try { 0; throw new Exception() } catch e { 0 ; 2 } then always { 0 }, 3))
```

Comment:
Two things:

- it took me a bit of time to realize what was `o.add()` doing. I'd rather try to avoid short variables and use more intention revealing names
- I would also prefer to have several tests with individual asserts, at least for the new tests, if it's not a nightmare